src/backoff.ts

```diff
@@ -81,7 +90,7 @@ export async function retry(
   options: BackoffOptions,
   targetFunction: any,
   thisArg: any = null,
-  ...args: any[],
+  ...args: any[] // tslint:disable-line
```

Comment:
tsc's `A rest parameter or binding pattern may not have a trailing comma.ts(1013)` was fighting with tslint's `trailing-comma` rule